OpenRA.Game/Network/Session.cs

```diff
@@ -90,11 +90,16 @@ public string FirstEmptyBotSlot()
 			return Slots.FirstOrDefault(s => !s.Value.Closed && ClientInSlot(s.Key) == null && s.Value.AllowBots).Key;
 		}
 
-		public bool IsSinglePlayer
+		public bool IsSingleHuman
 		{
 			get { return Clients.Count(c => c.Bot == null) == 1; }
 		}
 
+		public IEnumerable<Session.Client> HumanPlayers
```

Comment:
One thing at a time.  That affects external compatibility, so is a much bigger deal to change.